adafruit_platformdetect/board.py

```diff
@@ -336,6 +337,14 @@ def _armbian_id(self):
             return ORANGE_PI_PC
         return None
 
+    def _armbian_id(self):
```

Comment:
this code should be merged with the above, not replace!

Reply:
You mean like this?

def _armbian_id(self):
        """Check whether the current board is an OrangePi PC."""
        board_value = self.detector.get_armbian_release_field('BOARD')
        if board_value == "orangepipc":
            return ORANGE_PI_PC
	if board_value == "orangepi-r1":
            return ORANGE_PI_R1
        return None

Reply:
yep

Reply:
OK, changed